legend-engine-xt-relationalStore-pure/src/main/resources/core_relational/relational/protocols/pure/vX_X_X/invocations/execution_relational_testConnection.pure

```diff
@@ -151,7 +153,19 @@ function meta::protocols::pure::vX_X_X::transformation::toPureGraph::connection:
             // _type = 'gcpWorkloadIdentityFederationWithAWS',
             serviceAccountEmail = $b.serviceAccountEmail,
             additionalGcpScopes = $b.additionalGcpScopes
-         )
+         ),
+      a:meta::protocols::pure::vX_X_X::metamodel::store::relational::connection::alloy::authentication::AwsPKAuthenticationStrategy[1] |
```

Comment:
Can you please add an example model as a test ? What does the "user" attribute represent ?

Reply:
user refers to a username, let me fix that to be similar to our other flows. Can you elaborate on what you mean by adding an example model? Can you point me at something existing?